Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift

```diff
@@ -111,34 +118,54 @@ extension TestListViewController: EndpointDelegate {
                 case .cycle:
                     let tmpList = TestCycleListModel()
                     tmpList.extractCycleList(fromData: unwrappedData, parentId: self.selectedPlanId)
+                    //if there are no cycles, display an empty cycle with the default value set to No Cycles Found, made unclickable in the buildCycleCell function below
                     if tmpList.testCycleList.isEmpty {
-                        return
+                        let emptyCycle = TestCycleModel();
+                        tmpList.testCycleList.insert(emptyCycle, at: 0)
+                        
```

Comment:
Since one of the reasons that tmpList can be empty is if the cycles returned are not part of the currently selected plan I think we may need to change this logic a bit. maybe inside this if statement we could have an if totalItems != 0{return} else {...addEmptyCycleLikeNow...}. I think that it could be refactored a bit more but that should work. This is hard to test because you need to select a plan, then while all the cycles are page loaded select another plan.